recipes/psyinf-gmtl/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conan import ConanFile
+from conan.tools.files import export_conandata_patches, get, copy
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "psyinf-gmtl"
+    description = "The Generic Math Template Library. A math library designed to be high-performance, extensible, and generic."
+    license = "LGPL-2.1-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://ggt.sourceforge.net/html/main.html"
```

Comment:
This is a fork of the original

```suggestion
    homepage = "https://github.com/psyinf/gmtl"
```